draft-ietf-ohai-ohttp.md

```diff
@@ -249,7 +249,8 @@ Oblivious HTTP is more costly than a direct connection to a server.  Some costs,
 like those involved with connection setup, can be amortized, but there are
 several ways in which Oblivious HTTP is more expensive than a direct request:
 
-* Each request requires at least two regular HTTP requests, which adds latency.
+* Each request requires at least two regular HTTP requests, which adds
+  processing latency.
```

Comment:
I think it should be clear that the latency isn't due to extra round trips, but a "bump" in processing on the one round trip.

Reply:
But it could be due to extra round trips between intermediate hops on the path, right?

Reply:
Rather, there aren't any extra client-facing round trips.

Reply:
Right 👍